src/server/index.js

```diff
@@ -1,10 +1,10 @@
 const express = require('express');
 const app = express();
 const PORT = process.env.PORT || 3000;
-const checkDatabaseHealth = require('./database-check');
+const DatabaseHealth = require('./database-check');
 
 app.get('/heartbeat', function(req, res) {
-  checkDatabaseHealth(res);
+  DatabaseHealth.checkDatabaseHealth(res,DatabaseHealth.databaseResponse);
```

Comment:
it should work something like this:

```js
checkDatabaseHealth((isWorking) => {
  if (isWorking) {
    res.json(okStatus);
  } else {
    res.json(errorStatus);
  }
});
```

We can discuss this tomorrow in person.